platformcomponents/desktop/navigation-bar.json

```diff
@@ -0,0 +1,10 @@
+{
+  "navigation-bar": {
+    "comment": "This is applied to the first column navigation bar.",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/branch/WG6O99kWm33upbqMdnnIQA/Components---Windows%2BWeb?node-id=29743%3A57187",
+    "divider": {
+      "#normal": ["@theme-gradientdivider-normal-0", "@theme-gradientdivider-normal-1"],
+      "#hovered": ["@theme-gradientdivider-hover-0", "@theme-gradientdivider-hover-1"]
```

Comment:
the divider in the navigation bar stays with the same color for its three UI states (normal, hover and press). Thus, I would suggest simply saying:
"divider": ["@theme-gradientdivider-normal-0", "@theme-gradientdivider-normal-1"]